README.md

```diff
@@ -1,112 +1,53 @@
+Additionally, an HTML report is generated with information about the carbon footprint of the whole pipeline run and containing plots showing, for instance, an overview of the CO2 emissions for the different processes.
 
-- `plugins/nf-hello/src/resources/META-INF/MANIFEST.MF` 
-    
-    Manifest file defining the plugin attributes e.g. name, version, etc. The attribute `Plugin-Class` declares the plugin main class. This class should extend the base class `nextflow.plugin.BasePlugin` e.g. `nextflow.co2footprint.HelloPlugin`.
+## Quick Start
 
-- `plugins/nf-hello/src/resources/META-INF/extensions.idx`
-    
-    This file declares one or more extension classes provided by the plugin. Each line should contain the fully qualified name of a Java class that implements the `org.pf4j.ExtensionPoint` interface (or a sub-interface).
+Declare the plugin in your Nextflow pipeline configuration file:
 
-- `plugins/nf-hello/src/main` 
-
-    The plugin implementation sources.
-
-- `plugins/nf-hello/src/test` 
-
-    The plugin unit tests. 
-
-## Plugin classes
-
-- `HelloConfig`: simple example how to handle configuration options provided via the Nextflow configuration file. 
-
-- `HelloExtension`: show how create an extension class that can be used to create custom channel factories, operation and fuctions that can be imported in the pipeline script as DSL extensions.
-
-- `CO2FootprintFactory` and `CO2FootprintObserver`: show how to intercept workflow runtime events and react correspondly with custom code.
-
-- `HelloPlugin`: the plugin entry point.
-
-
-## Unit testing 
-
-Run the following command in the project root directory (ie. where the file `settings.gradle` is located):
-
-```bash
-./gradlew check
+```groovy title="nextflow.config"
+plugins {
+  id 'nf-co2footprint'
+}
 ```
 
-## Testing and debugging
-
-To run and test the plugin in for development purpose, configure a local Nextflow build with the following steps:
-
-1. Clone the Nextflow repository in your computer into a sibling directory:
-    ```bash
-    git clone --depth 1 https://github.com/nextflow-io/nextflow ../nextflow
-    ```
-  
-2. Configure the plugin build to use the local Nextflow code:
-    ```bash
-    echo "includeBuild('../nextflow')" >> settings.gradle
-    ```
-  
-   (Make sure to not add it more than once!)
-
-3. Compile the plugin alongside the Nextflow code:
-    ```bash
-    make compile
-    ```
-
-4. Run Nextflow with the plugin, using `./launch.sh` as a drop-in replacement for the `nextflow` command, and adding the option `-plugins nf-hello` to load the plugin:
-    ```bash
-    ./launch.sh run nextflow-io/hello -plugins nf-hello
-    ```
+This is all that is needed - Nextflow will automatically fetch the plugin code at run time.
 
-## Testing without Nextflow build
+Additionally, you can adjust the nf-co2footprint plugin parameters in your config file as follows:
```

Comment:
I would add this section under a different title. For example `Customising parameters`. And briefly describe each of the parameters that can be provided with the config file.

Reply:
Yes I see, but I actually didn't want to provide a complete documentation yet, since I do not consider it ready to use and we would need to check the handling of a few of those parameters. So I would postpone this a bit. What do you think?

Reply:
I think it will be confusing if people read this section and starts asking questions about the parameters, so I would prefer to write a short description. Otherwise, if you think it isn't ready, I would remove them from the README. The only one which will need more development is `country` right? Maybe we can just delete this one for now?